packages/northstar-wrapper/package.json

```diff
@@ -3,36 +3,43 @@
   "version": "1.5.1-beta.4",
   "private": "true",
   "description": "Package to wrap the Fluent NorthStar UI library, for short-term React 18 compatibility",
+  "module": "dist/dist-esm/index.js",
```

Comment:
I guess we will generate both esm and cjs code in our npm package? How much bigger our npm packages will be in the end

Reply:
Yea... I agree with production build should always use esm, but still there would be some app living in cjs world for some legacy reasons, so we might still want to support that well

For npm package, that is just a question out of curiosity, I did receive a feedback from webchat team asking about the  npm package size, but it shouldn't matter too much in the end

Reply:
True, and we do support cjs, we just don't optimize any bundles. However anyone using this in cjs will actually receive a benefit because they don't have to rollup the entire n* library, the small pieces needed are now directly bundled by us so smaller package overall (unless of course they use both us and n* in their app and are bundling using cjs)